resources/glossary/machine-learning.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Machine Learning
+description: general introduction to how Machine Learning could be applied in the context of multi-agent simulations
+slug: machine-learning
+---
+
+## WHAT IS MACHINE LEARNING
+
+**Machine Learning** is a sub-field of Artificial Intelligence where algorithms learn from interacting with an **environment** or from a **training dataset**. Machine Learning is primarily used to make **predictions** using statistics, applying those statitical methods to business problems is the realm of **predictive analytics**. Most recent advances in Machine Learning have been driven by research in Deep Learning, which is the combination of Machine Learning and multi-layer [artificial neural networks](https://hash.ai/wiki/neural-nets).
+
+<p align="center">
+![Artificial Intelligence Machine Learning Deep Learning](https://i.imgur.com/xatMpJa.png)
+</p>
+
+## WHY USE MACHINE LEARNING
+
+The advantage of using Machine Learning instead of traditional Artificial Intelligence algorithms is that models can act and adapt to different possible environments or data without being explicitly programmed to do so. In that sense, it is more flexible than rule-based Artificial Intelligence where humans often have to hard-code expert knowledge about the task at hand. A striking example has been AlphaZero beating the best (overly engineered) chess programs without any prior rule or knowledge about chess.
+
+The two use-cases of machine learning are to either give a numeric answer to a question (eg. "what will be the price of bitcoin tomorrow"), which we call a **regression problem**, the simpler example being a linear regression (aka a straight line passing through the data). The second use-case is **classification** where the objective is to tell if a given input belongs to a particular group. A popular example is to determine whether in an image there is a cat or a dog.
+
+## TYPES OF MACHINE LEARNING
+
+The three most important approaches in machine learning are supervised learning, unsupervised learning and reinforcement learning. Other methods such as [data mining](https://hash.ai/wiki/data-mining), semi-supervised learning or feature learning make use of those three essential methods listed above. Although Deep Learning has been the most popular paradigm in Machine Learning in the past 5 years, traditionally machine learning has made extensive use of decision trees, support-vector machines, bayesian networks and genetic algorithms. 
```

Comment:
We should avoid making positionally relative references to content/information on-page (e.g. in this case saying 'listed above').

It's unlikely content is printed out/even more unlucky if then split (and thus 'beneath' would become 'overleaf'), but we can avoid the possibility entirely by rephrasing things. @mtrazzi Please have a stab at this

Reply:
To make the second sentence less scary/word-soupy we might also add "(defined below)" to the end of it.